deploy/global.scope/west.hub/americas.scope/device5.device.json

```diff
@@ -0,0 +1,3 @@
+connection.properties.desired.connecttooverwrite: {
```

Comment:
Is this change interntional?

Reply:
yes, per morning sync, this is an example of what we would tackle in v2